railties/lib/rails/generators/erb/controller/controller_generator.rb

```diff
@@ -7,18 +7,26 @@ module Generators # :nodoc:
     class ControllerGenerator < Base # :nodoc:
       argument :actions, type: :array, default: [], banner: "action action"
 
-      def copy_view_files
-        base_path = File.join("app/views", class_path, file_name)
+      def create_root_directory
         empty_directory base_path
+      end
 
+      def create_view_files
         actions.each do |action|
           @action = action
+
           formats.each do |format|
             @path = File.join(base_path, filename_with_extensions(action, format))
             template filename_with_extensions(:view, format), @path
           end
         end
       end
+
+      private
+
```

Comment:
Can you remove this space? ✂️

Reply:
Thanks for the comment. I removed an empty line under `private` keyword.